spec/serializers/api/enterprise_shopfront_serializer_spec.rb

```diff
@@ -67,6 +67,5 @@
   it "serializes taxons" do
     expect(serializer.serializable_hash[:taxons]).to be_a ActiveModel::ArraySerializer
     expect(serializer.serializable_hash[:taxons].to_json).to match 'Meat'
-    expect(serializer.serializable_hash[:taxons].to_json).to match 'Veg'
```

Comment:
At first I was thinking "hey, there shouldn't be multiple taxons anymore."
But a shopfront has multiple products, so of course continues to have multiple taxons 👍